sonar-text-dotnet/src/SonarLint.Secrets.DotNet.UnitTests/Rules/AlibabaCloudAccessKeySecretsRuleTest.cs

```diff
@@ -68,7 +68,7 @@ public void TestRuleRegexNegative()
         {
             var testSubject = new AlibabaCloudAccessKeySecretsRule();
 
-            var secrets = testSubject.Find(readFileAndNormalize("src/test/files/google-cloud-account-key/GoogleCloudAccountNegative.json", UTF_8));
+            var secrets = testSubject.Find(readFileAndNormalize("..\\sonar-text-plugin\\src\\test\\resources\\checks\\GoogleCloudAccountKeyCheck\\GoogleCloudAccountNegative.json", UTF_8));
```

Comment:
We can create a constants class for the test projects and add a `Root` constant. Then use the constant in the tests. In this way, next time the root changes, we will only have to change one file.

Reply:
Maybe it's better to apply this change in another PR after we are done with the pipeline.

WDYT?
In case you agree I will open a git issue.

Reply:
I was thinking to do this as part of PR as we modify all the paths. It looks to me directly related.